src/network/core/network_game_info.cpp

```diff
@@ -149,7 +150,8 @@ const NetworkServerGameInfo *GetCurrentNetworkServerGameInfo()
 	 */
 	_network_game_info.companies_on  = (byte)Company::GetNumItems();
 	_network_game_info.spectators_on = NetworkSpectatorCount();
-	_network_game_info.game_date     = TimerGameCalendar::date;
+	_network_game_info.calendar_date = TimerGameCalendar::date;
+	_network_game_info.ticks_running  = TimerGameTick::counter;
```

Comment:
I know you don't care about nice alignment, but using two spaces doesn't align with anything 😛 

```suggestion
	_network_game_info.ticks_running = TimerGameTick::counter;
```